src/main/java/com/coherent/training/selenium/kapitsa/web/pages/seleniumeasy/AlertsPage.java

```diff
@@ -30,12 +38,12 @@ public Alert callPromptAlert(){
     public WebElement acceptAlert(){
         confirmAlert(confirmAlertButton);
 
-        return find(By.id("confirm-demo"));
+        return driver.findElement(By.id("confirm-demo"));
     }
 
     public WebElement dismissAlert(){
         declineAlert(confirmAlertButton);
 
-        return find(By.id("confirm-demo"));
+        return driver.findElement(By.id("confirm-demo"));
```

Comment:
1) why do we need finElement calls if we have possibility to instantiate them via factory?
2) if we found the element that does not means that we did something to element